src/Office-Add-in-ASPNET-SSO-WebAPI/App_Start/Startup.Auth.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Owin;
+using Microsoft.IdentityModel.Tokens;
+using System.Configuration;
+using Microsoft.Owin.Security.OAuth;
+using Microsoft.Owin.Security.Jwt;
+using Office_Add_in_ASPNET_SSO_WebAPI.App_Start;
+
+namespace Office_Add_in_ASPNET_SSO_WebAPI
+{
+	public partial class Startup
+	{
+		public void ConfigureAuth(IAppBuilder app)
+		{
+            TokenValidationParameters tvps = new TokenValidationParameters
+            {
+				ValidAudience = ConfigurationManager.AppSettings["ida:Audience"],
+				ValidIssuer = ConfigurationManager.AppSettings["ida:Issuer"],
```

Comment:
This line makes the add-in only valid for a single Tenant, which is not the case with node.js sample. Maybe we can just say validate the issuer is Microsoft?
				IssuerValidator = (issuer, token, tvp) =>
				{
					if (issuer.StartsWith("https://login.microsoftonline.com/"))
						return issuer;
					else
						throw new SecurityTokenInvalidIssuerException("Invalid issuer");
				},